frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-builder/task-sidebar/temp-utils.tsx

```diff
@@ -30,8 +30,10 @@ export const StringParam: React.FC<ParameterProps> = (props) => {
   const { currentValue, defaultValue, isValid, name, onChange, setDirty } = props;
 
   return (
-    <TextInput
+    <TextArea
       id={name}
+      rows={1}
+      style={{ minHeight: 30, resize: 'vertical' }}
```

Comment:
Hmmm... 30px is weird. We should probably think of a more deterministic value... but after a few tries off the top of my head I'm at a bit of a loss what that would be...

- Font-size is 14px (aka 1em)
- The padding + borders gives us 12px
- Adding those two together, gets us 26 -- which feels too tight 😞 
- LineHeight is set to 21px, so instead of font-size we use that, that's 33px -- which is about what you got.

Doing a calc of all the Patternfly variables to get to the 33px turns out to be an ugly slapping together of things. I think we can probably go with the static value for now -- unlikely to cause us too much headache I would think. Sizes tend not to change too much in this area. If they say, change padding, we'll end up in an awkward height situation.

Reply:
updated the height to 33px